src/js/utils/ui.js

```diff
@@ -378,7 +378,12 @@ function removeHandlers(ui, triggerName) {
     const listenerOptions = options[triggerName];
     if (listeners) {
         Object.keys(listeners).forEach(domEventName => {
-            element.removeEventListener(domEventName, listeners[domEventName], listenerOptions);
+            const useCapture = listenerOptions[domEventName];
+            if (typeof useCapture === 'boolean') {
+                element.removeEventListener(domEventName, listeners[domEventName], useCapture);
+            } else {
+                element.removeEventListener(domEventName, listeners[domEventName]);
```

Comment:
There's a note on MDN 

> It's worth noting that some browser releases have been inconsistent on this, and unless you have specific reasons otherwise, it's probably wise to use the same values used for the call to addEventListener() when calling removeEventListener().

should we instead always specify the options ?

Reply:
The bug was caused by us passing in an object. IE11 will treat an object as `true` and then not remove the listener. That is the bug that this change fixes.